packages/react-examples/src/date-time/Calendar/Calendar.doc.tsx

```diff
@@ -0,0 +1,108 @@
+const CalendarInlineWeekNumbersExampleCode = require('!raw-loader!@fluentui/react-examples/src/date-time/Calendar/Calendar.Inline.WeekNumbers.Example.tsx') as string;
+const CalendarInlineWeekSelectionExampleCode = require('!raw-loader!@fluentui/react-examples/src/date-time/Calendar/Calendar.Inline.WeekSelection.Example.tsx') as string;
+const CalendarInlineMonthSelectionExampleCode = require('!raw-loader!@fluentui/react-examples/src/date-time/Calendar/Calendar.Inline.MonthSelection.Example.tsx') as string;
+
+export const CalendarPageProps: IDocPageProps = {
+  title: 'Calendar',
+  componentName: 'Calendar',
+  componentUrl: 'https://github.com/microsoft/fluentui/tree/master/packages/date-time/src/components/Calendar',
+  examples: [
+    {
+      title: 'Calendar',
+      code: CalendarInlineExampleCode,
+      view: <CalendarInlineExample />,
+    },
+    {
+      title: 'Calendar with overlaid month picker when header is clicked',
+      code: CalendarInlineOverlaidMonthExampleCode,
+      view: <CalendarInlineOverlaidMonthExample />,
+    },
+    {
+      title: 'Calendar with week selection',
+      code: CalendarInlineWeekSelectionExampleCode,
+      view: <CalendarInlineWeekSelectionExample />,
+    },
+    {
+      title: 'Calendar with month selection',
+      code: CalendarInlineMonthSelectionExampleCode,
+      view: <CalendarInlineMonthSelectionExample />,
+    },
+    {
+      title: 'Calendar with week numbers',
+      code: CalendarInlineWeekNumbersExampleCode,
+      view: <CalendarInlineWeekNumbersExample />,
+    },
+    {
+      title: 'Calendar displaying 6 weeks',
+      code: CalendarInlineSixWeeksExampleCode,
+      view: <CalendarInlineSixWeeksExample />,
+    },
+    {
+      title: 'Calendar with month picker and no day picker',
+      code: CalendarInlineMonthOnlyExampleCode,
+      view: <CalendarInlineMonthOnlyExample />,
+    },
+    {
+      title: 'Calendar with date boundary and disabled dates',
+      code: CalendarInlineDateBoundariesExampleCode,
+      view: <CalendarInlineDateBoundariesExample />,
+    },
+    {
+      title: 'Calendar with custom selectable days',
+      code: CalendarInlineContiguousWorkWeekDaysExampleCode,
+      view: <CalendarInlineContiguousWorkWeekDaysExample />,
+    },
+    {
+      title: 'Calendar with custom selectable days, week starting on Monday',
+      code: CalendarInlineNonContiguousWorkWeekDaysExampleCode,
+      view: <CalendarInlineNonContiguousWorkWeekDaysExample />,
+    },
+    {
+      title: 'Calendar with multi-day view',
+      code: CalendarInlineMultidayDayViewExampleCode,
+      view: <CalendarInlineMultidayDayViewExample />,
+    },
+    {
+      title: 'Calendar a tooltip for each day and disabling weekends',
+      code: CalendarInlineCustomDayCellRefExampleCode,
+      view: <CalendarInlineCustomDayCellRefExample />,
+    },
+    {
+      title: 'Calendar launched from a button',
```

Comment:
The examples in the new version are based on the set from the date-time package that was converted to use discrete examples for each concept rather than a mega-example with props (I think @lorejoh12 did those conversions?). I also simplified some of the names. It looks like we have 13 now compared to 15 before, but if there's anything important missing we can add it back later.